pieces.py

```diff
@@ -10,28 +10,52 @@ class Pawn(Piece):
 	def __init__(self, is_white, square):
 		super().__init__(is_white, square)
 		self.movement_type = ("forward", "backward", "forward_left", "forward_right", "backward_left", "backward_right")
+		if is_white == True:
+			self.piece_name = "WP" 
+		else:
+			self.piece_name = "BP"
```

Comment:
Since the object already knows its piece type (from its class) and its color (from the `is_white` property), is there a reason to save `piece_name` as a property here, rather than having it be a method that can calculate this dynamically when needed?

Reply:
No, this is a good point. This is a result of me including piece type in the class definition after I had already included this code and not refactoring it. Will fix.